LondonBritishLibrary/orient/BLorient3663.xml

```diff
@@ -0,0 +1,152 @@
+                            <supportDesc>
+                                <support>
+                                    <material key="parchment"/>
+                                </support>
+                                <extent>
+                                    <measure unit="leaf">1</measure>
+                                    <dimensions type="outer" unit="mm">
+                                        <height>1735</height>
+                                        <width>90</width>
+                                    </dimensions>
+                                    <note>Composed of three strips.</note>
+                                </extent>
+                            </supportDesc> 
+                        </objectDesc>
+                        <handDesc>
+                            <handNote xml:id="h1" script="Ethiopic">
+                                <desc>Large, mediocre handwriting.</desc>
+                                <date notBefore="1700" notAfter="1899" resp="PRS8999Strelcyn"/>
+                            </handNote>
+                        </handDesc>
+                        <decoDesc>
+                            <decoNote xml:id="d1" type="miniature">
+                                <desc>Coloured <term key="Magic">magic picture</term>.</desc>
+                            </decoNote>
+                            <decoNote xml:id="d2" type="miniature">
+                                <desc>Coloured <term key="Magic">magic picture</term>.</desc>
+                            </decoNote>
+                            <decoNote xml:id="d3" type="miniature">
+                                <desc>Coloured <term key="Magic">magic picture</term>.</desc>
+                            </decoNote>
+                        </decoDesc>
+                        <bindingDesc>
+                            <binding xml:id="binding">
+                                <decoNote xml:id="b1" type="SlipCase">Contained in a cylindrical case in <material key="leather">leather</material>.</decoNote>
+                            </binding>
+                        </bindingDesc>
+                    </physDesc>
+                    <history>
+                        <origin>
+                            <origDate notBefore="1700" notAfter="1899">18th-19th century (?)</origDate>
+                        </origin>
+                        <provenance>The name of the first owner <persName role="owner" ref="PRS14548WaldaAragawi">Walda ʾAragāwi</persName>,
+                            written in red, has been erased everywhere except in No. 2. The name of the second owner, written in black ink, illegibly,
+                            seems to be <persName role="owner" ref="PRS14549WalattaSamat">Walatta Samā<supplied reason="subaudible" resp="PRS8999Strelcyn">ʿ</supplied>t</persName>.</provenance>
+                        <acquisition>On the cover of the box containing the scroll: 'Presented by 
```

Comment:
I wonder if, though it is on the box, we should list this AcquisitionNote listed as an extra, then it would be a `q` with `xml:lang="en"`